src/Raven.Server/Utils/CpuUsage.cs

```diff
@@ -96,6 +113,24 @@ private static double CalculateProcessCpuUsage(ProcessInfo currentInfo, ProcessI
             return Math.Min(100, processCpuUsage);
         }
 
+        private static (double MachineCpuUsage, double ProcessCpuUsage) CalculateCpuFromExtensionPoint()
+        {
+            var extensionPoint = new ExtensionPointInfo(CpuUsageExtensionPoint.Data);
+            var machineCpuUsage = extensionPoint.MachineCpuUsage;
+            if (machineCpuUsage < 0)
```

Comment:
In what case this can happen? And why shouldn't we abort the extension at this point?

Reply:
@ayende This can happen when: 
* There were issues in the extension point process and there was one try to restart
*  If the process returns a negative value.
What do you mean by abort? Just to kill the process? In the first scenario, it is already the situation. In the second scenario if I check for bottom edge maybe to check the upper side?

Reply:
Yes, on bad input from the script, kill it and retry.

Reply:
https://github.com/ravendb/ravendb/pull/7960#discussion_r232561460